src/modal/modal.ts

```diff
@@ -46,6 +46,22 @@ export class CaptureEyeModal extends LitElement {
     if (setAsLoaded) this._assetLoaded = true;
   }
 
+  updateEngagementZone(
+    engagementImage: string,
+    engagementLink: string,
+    actionButtonText: string,
+    actionButtonLink: string
+  ) {
+    if (this.engagementImage !== engagementImage) {
+      this._imageLoaded = false;
+      this.engagementImage = engagementImage;
+      this.requestUpdate();
```

Comment:
Does line 66 need to be removed to compare the correct value?

Reply:
It turns out has no effect on the current UI so they seem to be able to be safely removed.